src/dispatch/common.rs

```diff
@@ -148,8 +148,12 @@ async fn check_permissions_and_cooldown_single<'a, U, E>(
         None => {}
     }
 
-    // Only continue if command checks returns true. First perform global checks, then command
-    // checks (if necessary)
+    // Only continue if command checks returns true or
+    // `FrameworkOptions::skip_checks_for_owners` is set to true
+    // First perform global checks, then command checks (if necessary)
+    if ctx.framework().options.skip_checks_for_owners {
+        return Ok(());
+    }
```

Comment:
Only custom checks and cooldowns are skipped, not guild, DM, NSFW, permission, and bot permission checks. Is this intended?

Reply:
actually forgot about that. where should I look for those checks?

Reply:
Look above 😄 It's all in that one function